src/lib/mocking/middleware-functions/ilpModel.js

```diff
@@ -38,6 +38,11 @@ const handleQuoteIlp = (context, response) => {
   // Check whether the request is POST /quotes
   const pathMatch = /\/quotes\/([^/]+)$/
   if (context.request.method === 'post' && response.method === 'put' && pathMatch.test(response.path)) {
+    if (response.eventInfo && response.eventInfo.params && response.eventInfo.params.body &&
```

Comment:
You are assuming here that if someone overrides either the ilpPacker or condition, then they will not want either to be generated by the mock?

Reply:
Yes that's right, if only one of those values specified, we can't calculate the other values. So I am just breaking the calculation function here.

Reply:
Lets say, if they specify only ilpPacket, then the condition will not be calculated by TTK and there will be random string in the mock.